Kajabi/sage-lib
# Upload Card Documentation Updates

## Description
Updates documentation and converts Upload Card to Sage component.

### Screenshots
No visual changes with these adjustments.

## Test notes

### Steps for testing
1. Visit http://localhost:4000/pages/breakout/object/upload_card
2. Verify UI and documentation updates.


## Related

- N/A
